tests/qb/qb-ics/ic-04-dimensions-have-range.sparql

```diff
@@ -1,12 +1,12 @@
-# Every dimension declared in a qb:DataStructureDefinition must have a declared rdfs:range.
```

Comment:
This comment is very useful. It's what gets displayed to us when a test fails.

Reply:
Noted. I'll add them back.

Reply:
@canwaf I did it!